modules/Twitter/src/main/nbm/manifest.mf

```diff
@@ -0,0 +1,5 @@
+Manifest-Version: 1.0
+OpenIDE-Module-Name: Twitter
+OpenIDE-Module-Short-Description: Twitter Stream
+OpenIDE-Module-Long-Description: Get Twitter Stream displayed
```

Comment:
These are the values that will appear for your plugin with Gephi and also within the plugin marketplace. I guess they are all right, but you might consider adding a little more detail (particularly to the Long Description). You might also make the Module Name a little more specific. For example,
```
OpenIDE-Module-Name: TwitterImporter
OpenIDE-Module-Short-Description: Plugin to create a graph from the Twitter Streaming API
OpenIDE-Module-Long-Description: <p>This plugin connects to the Twitter Streaming API and represents tweets as graph.</p><p>It includes two possible ways to represent a graph:<ul><li>User Network: A weighted User to User network with parallel edges for RT and Mentions</li><li>Full Smart Network: A full representation with nodes for Users, Tweets, Hashtags, Urls, Media and Symbols</li></p>